roles/core/repositories_client/readme.rst

```diff
@@ -6,6 +6,43 @@ Description
 
 This role simply configure repositories for client hosts.
 
+Instructions
+^^^^^^^^^^^^
+
+See repositories_server role instructions for general repositories path details.
+
+Repositories are set in **repositories** variable. Two shapes are available: a
+simple one and an advanced one.
+
+Simple one:
+
+.. code-block:: yaml
+
+  repositories:
+    - os
+    - bluebanquise
+    - myrepo
+
+Advanced one, to be combined with simple one as desired:
+
+.. code-block:: yaml
+
+  repositories:
+    - os
+    - bluebanquise
+    - name: epel
```

Comment:
While technically functional, @oxedions wanted to recommend the use of `external_repositories` for repositories hosted outside of the cluster.
No problem for me to deprecate `external_repositories` though.

Reply:
Ox is right, updating.

Reply:
Assuming we can deprecate external_repositories, makes sense here. Adding a task in the list.